docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration file is for the **development** setup.
 # For a production example please refer to setup/docker-compose.yml.
-version: '3'
+version: '3.7'
```

Comment:
Was this necessary?

Reply:
The [long volume syntax](https://docs.docker.com/compose/compose-file/#long-syntax-3), required in order to specify the volume type which was required to get watchdog to pick up host changes, was introduced in version 3.2. We could downgrade it to 3.2 if you prefer to support docker-compose 17.0.4, I was just following docs.